### PR TITLE
[civ2][8] move rllib algorithm, model to civ2

### PR DESCRIPTION
**Pull request by can-anyscale**
Move rllib algorithm + model to civ2. 

Test:
- CI

### CODE REVIEW

File: .buildkite/rllib.rayci.yml [hunk 1]
```diff
@@ -50,6 +142,27 @@ steps:
...
+        --only-tags examples
+        --except-tags multi_gpu,gpu 
+        --test-env RAY_USE_MULTIPROCESSING_CPU_COUNT=1
+        --skip-ray-installation # reuse the same docker image as the previous run
+
+      # rlmodule tests
+      - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib --run-flaky-tests --parallelism-per-worker 3
+        --only-tags rlm
+        --test-env RLLIB_ENABLE_RL_MODULE=1
+        --test-env RAY_USE_MULTIPROCESSING_CPU_COUNT=1
```
Comment (aslonnie) [inline .buildkite/rllib.rayci.yml:164]: why do these all run in one job?

**Review by aslonnie — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/parksandrec/">Parks and Recreation</a> on <a href="https://giphy.com/gifs/parksandrec-episode-3-parks-and-recreation-rec-5cuiY8a99aA9oBaIpU">GIPHY</a></div>

**Comment by aslonnie** [comment]
does @kouroshHakha review these PR's?

**Comment by can-anyscale** [comment]
Is Kourosh also an LLM person today ;). I'm tagging other folks in rllib team @avnishn, @ArturNiederfahrenhorst as an FYI of these changes

**Pull request merged**